flocker/node/agents/ebs.py

```diff
@@ -0,0 +1,198 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+An EBS implementation of the ``IBlockDeviceAPI``.
+"""
+
+from bitmath import Byte, GB
+import time
+from uuid import UUID
+
+from boto import ec2
+from pyrsistent import PRecord, field
+from zope.interface import implementer
+
+from .blockdevice import IBlockDeviceAPI, BlockDeviceVolume, UnknownVolume
+
+DATASET_ID_LABEL = u'flocker-dataset-id'
+METADATA_VERSION_LABEL = u'flocker-metadata-version'
+CLUSTER_ID_LABEL = u'flocker-cluster-id'
+
+
+def ec2_client(region, zone, access_key_id, secret_access_key):
+    """
+    Establish connection to EC2 client.
+
+    :param String region: The name of the EC2 region to connect to.
```

Comment:
I think the type here and below should be `str` or `unicode`....`unicode` probably.